src/goto-programs/string_instrumentation.h

```diff
@@ -15,16 +15,21 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "goto_functions.h"
 
 class message_handlert;
+class goto_modelt;
 
 void string_instrumentation(
-  symbol_tablet &symbol_table,
-  message_handlert &message_handler,
-  goto_programt &dest);
+  symbol_tablet &,
+  message_handlert &,
+  goto_programt &);
 
 void string_instrumentation(
-  symbol_tablet &symbol_table,
-  message_handlert &message_handler,
-  goto_functionst &dest);
+  symbol_tablet &,
```

Comment:
Should be done globally, i.e., all symbol_tablet/goto_functionst variants should eventually go away.